impeller/core/device_buffer.cc

```diff
@@ -22,21 +22,6 @@ BufferView DeviceBuffer::AsBufferView(std::shared_ptr<DeviceBuffer> buffer) {
   return view;
 }
 
-std::shared_ptr<Texture> DeviceBuffer::AsTexture(
```

Comment:
We don't actually use this anywhere, and the implementation for iOS creates a linear texture which is probably not what we want

Reply:
Also this impelementation fails on the Simulator.

Reply:
sgtm